nbautoexport/__init__.py

```diff
@@ -1,4 +1,8 @@
 from nbautoexport.export import post_save
-from nbautoexport.utils import __version__
+from nbautoexport.utils import __version__, get_logger
 
-__all__ = [post_save, __version__]
+__all__ = [
+    "post_save",
+    "get_logger",
+]
+__version__
```

Comment:
`__all__` is actually meant to be what you configure to be imported when a user does `from nbautoexport import *`. It's been a secondary usage to satisfy the `Module imported but unused (F401)` flake8 rule.

I just have `__version__` out here separately so that it also satisfies F401 but isn't part of the set of things that should be imported by `*`.